src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/TidesController.java

```diff
@@ -2,8 +2,43 @@
 
 import org.springframework.web.bind.annotation.RestController;
 
+import edu.ucsb.cs156.spring.backenddemo.services.TidesQueryService;
+import lombok.extern.slf4j.Slf4j;
 
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+
+@Api(description="Tides from https://api.tidesandcurrents.noaa.gov/api/prod/")
```

Comment:
I'm not sure how strict we have to be with the API documentation, but just to be safe, maybe we should match the staff example exactly.  Their description is:
`Tide Information from NOAA https://api.tidesandcurrents.noaa.gov/api/prod/`

Reply:
@eduglt fixed!